tests/drivers/gpio/gpio_basic_api/testcase.yaml

```diff
@@ -16,3 +16,7 @@ tests:
     platform_allow: nrf52840dk_nrf52840 nrf52_bsim
     extra_args: "DTC_OVERLAY_FILE=boards/nrf52840dk_nrf52840.overlay;\
                  boards/nrf52840dk_nrf52840_sense_edge.overlay"
+
+  drivers.gpio.mr_canhubk3_wkpu:
+    platform_allow: mr_canhubk3
+    extra_args: "DTC_OVERLAY_FILE=boards/mr_canhubk3_wkpu.overlay"
```

Comment:
Why is this needed. Can we delete this?

Reply:
this is adding a new test scenario for the `mr_canhubk3` board to test with WKPU interrupts. The default test scenario is to test with EIRQ interrupts:
https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/gpio/gpio_basic_api/boards/mr_canhubk3.overlay#L16-L21